src/command/format_stdin.rs

```diff
@@ -21,18 +22,6 @@ pub fn format_stdin_cmd(
         }
     };
 
-    // Search for the treefmt.toml from there.
-    let config_file = match config::lookup(work_dir) {
-        Some(path) => path,
-        None => {
-            return Err(anyhow!(
-                "{} could not be found in {} and up. Use the --init option to create one.",
-                config::FILENAME,
-                work_dir.display()
-            ));
-        }
```

Comment:
This check is in `run_cli()` and now also applies to both `format_cmd()` and `format_stdin_cmd()`.